adapter/config/default_config.go

```diff
@@ -116,6 +116,12 @@ var defaultConfig = &Config{
 				IdleTimeoutInSeconds:           3600,
 			},
 		},
+		PayloadPassing: payloadPassing{
```

Comment:
```suggestion
		PassingPayloadToEnforcer: passingPayloadToEnforcer{
```
Looks like you have missed this. @pubudu538 raised this during the code review AFAIR. If we didn't mention `ToEnforer`, a user might think that the three values `MaxRequestBytes`, `AllowPartialMessage`, `PackAsBytes` affects the payload that goes to the backend too.